docs/_documentations/vsc-codechange.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-title: "Getting started with Codewind for VS Code"
-description: "Getting started with Codewind for VS Code"
+title: "Making a code change"
+description: "Making a code change"
```

Comment:
If all the IDEs will have a file of the same title, I'm thinking we should add the name of the IDE as part of the title. Adding the name could also help orient readers to what's going on in this file if they happen to see this page out of context from the rest of the table of contents.

Maybe something like..."Making a code change with VS Code"?